modules/page_base.py

```diff
@@ -533,6 +533,51 @@ def context_click(
             el = self.fetch(reference, labels)
             self.actions.context_click(el).perform()
         return self
+    
+    def paste_to_element(
+        self, sys_platform, reference: Union[str, tuple, WebElement], labels=[]
+    ) -> Page:
+        """ Paste the copied item into the element"""
+        with self.driver.context(self.context_id):
+            el = self.fetch(reference, labels)
+            self.scroll_to_element(el)
+            self.actions.click(el).perform()
+            if sys_platform == "Darwin":
```

Comment:
This is fine, but you can factor out the key with

```Python
mod_key = Keys.COMMAND if sys_platform == "Darwin" else Keys.CONTROL
self.actions.key_down(mod_key)
...
```

Reply:
changing to this as well